Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -8,23 +8,49 @@ import UIKit
 internal final class CommentsViewController: UITableViewController {
   // MARK: - Properties
 
+  fileprivate let dataSource = CommentsDataSource()
   fileprivate let viewModel: CommentsViewModelType = CommentsViewModel()
 
   // MARK: - Lifecycle
 
   internal override func viewDidLoad() {
     super.viewDidLoad()
 
+    self.navigationItem.title = Strings.project_menu_buttons_comments()
+
+    self.tableView.dataSource = self.dataSource
+
+    self.dataSource.load()
+
     self.viewModel.inputs.viewDidLoad()
+
+    self.tableView.registerCellClass(CommentCell.self)
+    self.tableView.registerCellClass(CommentPostFailedCell.self)
+    self.tableView.registerCellClass(CommentRemovedCell.self)
+  }
+
+  internal override func viewWillAppear(_ animated: Bool) {
+    super.viewWillAppear(animated)
+    self.navigationController?.setNavigationBarHidden(false, animated: animated)
```

Comment:
Do we want this hidden?

Reply:
Yes otherwise we won't see the NavBar.

Reply:
oh sorry missed that it's being set to `false`.... ok, will need to just test that this doesn't affect other areas.